### PR TITLE
chore: add draft api changes for webauthn based assertion

### PR DESCRIPTION
**Pull request by kleyow**

### CODE REVIEW

File: docs/thirdparty-openapi3-snippets.yaml [hunk 1]
```diff
@@ -69,6 +69,7 @@ paths:
                         - OTP
                         - QRCODE
                         - U2F
+                        - PKC
```
Comment (eoln) [inline docs/thirdparty-openapi3-snippets.yaml:72]: do we really need PKC?

File: thirdparty/openapi3/components/schemas/AuthenticationValue.yaml [hunk 2]
```diff
@@ -3,6 +3,7 @@ anyOf:
   - $ref: '../../../../fspiop/v1_1/openapi3/components/schemas/OtpValue.yaml'
   - $ref: '../../../../fspiop/v1_1/openapi3/components/schemas/QRCODE.yaml'
   - $ref: ./U2FPinValue.yaml
+  - $ref: ./FIDOPublicKeyCredential.yaml
```
Comment (eoln) [inline thirdparty/openapi3/components/schemas/AuthenticationValue.yaml:6]: FIDOPublicKeyCredential has a payload for attestation  (credential registration)
for assertion, we have little different payload:
```
{
    id: 'ADSUllKQmbqdGtpu4sjseh4cg2TxSvrbcHDTBsv4NSSX9...',
    rawId: ArrayBuffer(59),
    response: AuthenticatorAssertionResponse {
        authenticatorData: ArrayBuffer(191),
        clientDataJSON: ArrayBuffer(118),
        signature: ArrayBuffer(70),
        userHandle: ArrayBuffer(10),
    },
    type: 'public-key'
}
```

Comment (eoln) [reply thirdparty/openapi3/components/schemas/AuthenticationValue.yaml:6]: there must be `signature` & `userHandler`

**Review by eoln — CHANGES_REQUESTED** [review]

File: thirdparty/openapi3/components/schemas/AuthenticationValue.yaml (hunk 2 above)
Comment (kleyow) [reply thirdparty/openapi3/components/schemas/AuthenticationValue.yaml:6]: Ok I'll split FIDOPublicKeyCredential into FIDOPublicKeyCredentialAttestation and FIDOPublicKeyCredentialAssertion and make the changes.

File: docs/thirdparty-openapi3-snippets.yaml (hunk 1 above)
Comment (kleyow) [reply docs/thirdparty-openapi3-snippets.yaml:72]: nope. err'd too much on the side of caution and defaulted to addition.

**Comment by kleyow** [comment]
@lewisdaly You'll have to admin merge it as well since Pawel had changes requested.

**Review by lewisdaly — APPROVED** [review]

**Pull request merged**